src/main/java/friday/model/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package friday.model;
+
+import friday.logic.Parser;
+import java.time.LocalDateTime;
+
+/**
+ * A {@link friday.model.Task} with a deadline.
+ */
+public class Deadline extends Task {
+    final LocalDateTime due;
+    public Deadline(String d, LocalDateTime due) {
+        super(d);
+        this.due = due;
+    }
+    String typeIcon() {
```

Comment:
Should the method name contain a verb? 
```suggestion
    String getTypeIcon() {
```